dbfit-java/build.gradle

```diff
@@ -222,3 +251,106 @@ project('netezza') {
     }
 }
 
+// Aggregated dependencies to include in the POM file when publishing.
+dependencies {
+    compile publicCoreCompileDeps
+    runtime publicHsqldbRuntimeDeps
+    compile publicOracleCompileDeps
+    compile publicMysqlCompileDeps
+    compile publicPostgresCompileDeps
+}
+
+// Remove the default artifact as there are no Java sources for this project.
+configurations.archives.artifacts.clear()
+
+task allJavadoc(type: Javadoc) {
+    source subprojects.collect { project ->
+        project.sourceSets.main.allJava
+    }
+    destinationDir = new File(buildDir, 'doc/javadoc')
+    classpath = files(subprojects.collect { project ->
```

Comment:
What about using the default `it` param here and in the `source` (the definitions would be able to fit in single lines).